torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1509,12 +1523,24 @@ def _compiled_autograd_key(ctx):
 
             @staticmethod
             def forward(ctx, *deduped_flat_tensor_args):
+                if torch._functorch.config.aotd_debug_profile:
+                    get_chromium_event_logger().log_event_start(
```

Comment:
did you sanity check that `get_chromium_event_logger().log_event_start()` is fast enough that it doesn't throw off our measurements? (especially since these runtime overhead benchmarks are in the range of micro-to-milliseconds, while most compile time benchmarks are seconds-to-minutes)